test/test.py

```diff
@@ -1248,9 +1248,6 @@ def setUp(self):
         path = ":".join([os.path.join(BASE_DIR, "scripts"), TEST_DIR, "$PATH"])
         sh.run_command("export PATH={0}".format(path))
         sh.run_command("export PYTHONPATH={0}".format(BASE_DIR))
-        if self.repl_provider == bash_repl:
-            # Disable the "python" module provided by bash-completion
-            sh.run_command("complete -r python python2 python3")
```

Comment:
This was not actually doing anything, since `self.repl_provider` is a separate method that calls `bash_repl`. Since the tests were still passing, I'm assuming this is no longer needed.